forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -43,7 +44,7 @@ object ForageSDK : ForageSDKApi {
             ).tokenizeCard(
                 cardNumber = currentEntry.getPanNumber()
             )
-            else -> ForageApiResponse.Failure("Invalid PAN entry")
+            else -> ForageApiResponse.Failure(400, listOf(ForageError(400, "bad_request", "Invalid PAN entry")))
```

Comment:
"bad_request" is not a valid code from [this list](https://docs.joinforage.app/docs/error-codes). Let's try to be close to that or at least use something from [the backend list](https://github.com/teamforage/payments-api/blob/854844c3a97740d4e32491d5ddf8e29e4006ecb5/api/error_response.py#L162).

Reply:
This comment applies in numerous places where you manually construct a `ForageApiResponse.Failure`